scripts/xtensa-build-all.sh

```diff
@@ -8,87 +8,64 @@ BUILD_ROM=no
 BUILD_DEBUG=no
 BUILD_FORCE_UP=no
 BUILD_JOBS=$(nproc --all)
-BUILD_JOBS_NEXT=0
 BUILD_VERBOSE=
 
 PATH=$pwd/local/bin:$PATH
 
-pwd=`pwd`
+pwd=$(pwd)
+
+print_usage()
+{
+    cat <<EOF
+usage: xtensa-build.sh [options] platform(s)
+       [-r] Build rom (gcc only)
+       [-a] Build all platforms
+       [-u] Force UP ARCH
+       [-d] Enable debug build
+       [-c] Configure defconfig
+       [-k] Use private key
+       [-v] Verbose Makefile log
+       [-j n] Set number of make build jobs. Jobs=#cores when no flag. \
+Inifinte when not specified.
```

Comment:
I have also seen this typo, but since we are just now refactoring the old code this can be done in a separate PR. But now that you pointed it I think we can add a separate patch in this PR.
@crazoes Remeber that in a PR like this fixes come first, then the actual refactoring.

Reply:
I like to treat typos the same as indentation/whitespace: don't obscure actual changes with them. However if that line changes ANYWAY in git for some other and good reason then it's a good opportunity to fix typos at the same time.

Reply:
@marc-hb @dbaluta I have maintained the order here, fixing first and then moving the code around. 

In the process I have noticed that when we order moving of code after fixes, we will always end up with merge conflicts after editing the fix commit. So what is the rationale behind ordering fixes first and refactors later?